data/rulesets.js

```diff
@@ -783,11 +776,13 @@ exports.BattleFormats = {
 				let types = template.types;
 				let baseTemplate = dex.getTemplate(template.baseSpecies);
 				if (template.prevo) types = types.concat(dex.getTemplate(dex.getTemplate(template.prevo).prevo || template.prevo).types);
-				for (let i in baseTemplate.otherFormes) {
-					let forme = dex.getTemplate(baseTemplate.otherFormes[i]);
-					if (baseTemplate.otherFormes && !forme.battleOnly) {
-						if (forme.forme !== 'Alola' && forme.forme !== 'Alola-Totem' && forme.baseSpecies !== 'Wormadam') {
-							types = types.concat(forme.types).concat(baseTemplate.types);
+				if (baseTemplate.otherFormes) {
+					for (const formeid of baseTemplate.otherFormes) {
+						let forme = dex.getTemplate(formeid);
+						if (baseTemplate.otherFormes && !forme.battleOnly) {
```

Comment:
The `baseTemplate.otherFormes &&` part seems redundant.

Reply:
I forgot to remove that when i moved the if statement out of the loop, oops